spec/routes/spec_helper.rb

```diff
@@ -17,6 +17,16 @@ def app
     Clover.freeze.app
   end
 
+  config.define_derived_metadata(file_path: %r{\A\./spec/routes/api/}) do |metadata|
+    metadata[:clover_api] = true
+  end
+
+  config.before do |example|
+    next unless example.metadata[:clover_api]
+    header "Content-Type", "application/json"
+    header "Accept", "application/json"
+  end
```

Comment:
We can push it down some more, but not sure I can elide the check, because the rspec config is global right? If you aggregate route tests and other tests together, the before block will run for all.

What push down might accomplish is skipping metadata creation (and the hook) when no api route file is in the test batch.

Reply:
okay, I pushed it down, but I still can't drop the `next` expression since that hook gets installed globally. Still, a tiny win for tests that do not include api components, e.g. to run one example or one of the numerous files not in API :)